README.md

```diff
@@ -425,13 +425,13 @@ by AdaptiveCpp.
 cd build
 export CC=[path/to/system/clang]
 export CXX=[path/to/AdaptiveCpp/install/bin/syclcc]
```

Comment:
Running the tests, `syclcc` explicitly says it is deprecated and it suggests to use `acpp`. An AdaptiveCpp user should know about it, but I suggest to update this path:
```suggestion
export CXX=[path/to/AdaptiveCpp/install/bin/acpp]
```